referencia/webpay/README.md

```diff
@@ -1446,7 +1446,7 @@ authorizationCode  <br> <i> xs:string </i> | Código de autorización de la tr
 creditCardType  <br> <i> creditCardType </i> | Indica el tipo de tarjeta que fue inscrita por el cliente (Visa, AmericanExpress, MasterCard, Diners, Magna).
 last4CardDigits  <br> <i> xs:string </i> | Los últimos 4 dígitos de la tarjeta usada en la transacción.
 transactionId  <br> <i> xs:long </i> | Identificador único de la transacción de pago.
-responseCode  <br> <i> xs:int </i> | Código de retorno del proceso de pago, donde: <br> 0 (cero) es aprobado. <br> -1, -2, -3, -4, -5, -6, -7, -8: Rechazo <br> -97: Límites Oneclick, máximo monto diario de pago excedido. <br> -98: Límites Oneclick, máximo monto de pago excedido <br> -99: Límites Oneclick, máxima cantidad de pagos diarios excedido.
+responseCode  <br> <i> xs:int </i> | Código de retorno del proceso de pago, donde: <br> 0 (cero) es aprobado. <br> -1, -2, -3, -4, -5, -6, -7, -8: Rechazo <br> -96: Se llama el método sin previamente realizar la autenticación bancaria.  <br> -97: Límites Oneclick, máximo monto diario de pago excedido. <br> -98: Límites Oneclick, máximo monto de pago excedido <br> -99: Límites Oneclick, máxima cantidad de pagos diarios excedido.
```

Comment:
I'm not sure I understand the explanation (If by "autenticacion bancaria" you mean calling `initInscription` I would be explicit that the problem is that `initInscription` wasn't called. But if that's the case, how can the merchant have a `tbkUser` value which can only be obtained via `finishInscription`?)

Reply:
Your're right this error is not  for in the authorization, this happen when the `finishInscription` is called without the banking authorization. I will fix it